db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Provisioning/StateMachines/Methods.class/__methods__/provision.rb

```diff
@@ -0,0 +1,19 @@
+#
+# Description: This method launches an Ansible job template
+#
+
+$evm.log("info", "Starting Ansible Tower Provisioning")
+
+task = $evm.root["service_template_provision_task"]
+service = task.destination
+template = service.job_template
+
+begin
+  job = service.launch_job
```

Comment:
@mkanoor yes we need. This is the first version, only works for the case no additional changes are made from user because our dialog for user input is not ready yet.